docs/plugins/data.malware-scan.md

```diff
@@ -0,0 +1,26 @@
+data.malware-scan
+==========
+
+This plugin extracts hashs(md5,sha1,sha256) of each attachment and feeds them to MHR based blacklists such as 
+[MALWARE HASH REGISTRY][1].
```

Comment:
more will coming soon, need answers from virustotal and opswat (metascan-online)